plugins/system76-launch/system76-launch.quirk

```diff
@@ -0,0 +1,4 @@
+# System76 Launch
+[DeviceInstanceId=USB\VID_3384&PID_0001&REV_0001]
+Plugin = system76_launch
+CounterpartGuid = USB\\VID_03EB&PID_2FF4
```

Comment:
I don't think you want the double slash there -- right? other files like plugins/modem-manager/modem-manager.quirk don't seem to include it. Otherwise LGTM now, thanks.

Reply:
Good point, it should not have the slash